executor/tests/test_local_session_handler.py

```
@@ -286,19 +305,88 @@ async def fake_wait_for_session_port(host, port, process):
+    """Input handling should survive PTY teardown races."""
+    from executor.modes.local.session_handler import LocalSession, LocalSessionHandler
+
+    class FailingTerminal:
+        def write(self, data):
+            raise OSError("pty closed")
+
+    handler = LocalSessionHandler(public_base_url="http://localhost:17888")
+    handler.sessions["terminal-1"] = LocalSession(
+        session_id="terminal-1",
+        session_type="terminal",
+        access_token="secret",
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to argument: "access_token"

(S106)